### ISSUE TITLE
Analytics report

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/research-data-harvester#8]
I am a PU staff member who needs to track who is accessing Princeton research datasets

**Comment by hackartisan** [comment]
The repositories I'm looking at so far display number of views and number of downloads on the info page for the data sets. I believe it's these analytics that are desired, as opposed to analytics from the discovery portal itself. @windcowles can you confirm?

**Comment by windcowles** [comment]
That’s correct. (We could get discovery portal analytics through whatever internal system that we presumably already have in place for our web traffic.)
